spec/models/user_spec.rb

```diff
@@ -121,7 +122,7 @@
       expect(user.google_image_url).to be_a String
       expect(user.zip_code).to be_a String
       expect(user.summary).to be_a String
-      expect(user.goal).to be_an Integer
+      expect(user.goal).to be_an String
```

Comment:
We weren't sure if this was right... but I think because it is stored in DB as integer, when we return it, it comes back as a string? We did test authing in from LocalHost with new form select values (from FE), and it does store correctly (no DB rollback), so I think this is right. @Scott-Borecki can you please take a look and confirm?

Reply:
@tvaroglu As discussed, that is correct.  The enum is stored in the DB as an integer and then Rails is able to interpret it as the string enum key.